tests/lib/linter/linter.js

```diff
@@ -1603,7 +1603,7 @@ describe("Linter", () => {
 
             assert.strictEqual(messages.length, 1);
             assert.strictEqual(messages[0].ruleId, "no-alert");
-            assert.strictEqual(messages[0].message, "Unexpected alert.");
+            assert.include(messages[0].message, "alert");
```

Comment:
or you can just use the `messageId`.

```suggestion
            assert.include(messages[0].messageId, "unexpected");
```

Reply:
Yeah. I figured the `messageId` would be sufficient for those tests, but didn't want to make a drastic change if there was some reason for testing the `message` text.

All of those are already checking the `messageId`. Should we remove the additional check for the `message`? Is there any reason to keep the second assertion?

Reply:
I think it's fine to just remove the message, if already having messageId.

Reply:
@aladdin-add updated.

Reply:
@aladdin-add Sorry, I misread and thought that we were talking about the `ruleId` being sufficient.

I have updated the tests to check for the `messageId` as requested.